src/control/server/server.go

```diff
@@ -62,6 +62,10 @@ func cfgHasBdev(cfg *Configuration) bool {
 	return false
 }
 
+func instanceShmID(idx int) int {
+	return os.Getpid() + idx + 1
```

Comment:
why +1?

Reply:
Somewhat arbitrary, just seemed intuitive to me that determining an instance's shmid could be determined by adding N to the daos_server pid, where N is the ordinal Nth instance (vs having an offset-based scheme). I put it into the function so that we could change the details without having to change callers, though.